### PR TITLE
chore: tests for Lazy and moving out of unstable

### PR DESCRIPTION
**Pull request by ruseinov**
work on https://github.com/near/near-sdk-rs/issues/1199

### CODE REVIEW

File: near-sdk/tests/test-contracts/lazy/src/lib.rs
Comment (akorchyn) [inline near-sdk/tests/test-contracts/lazy/src/lib.rs:53]: I know we use the same for store contracts but what's the motivation to make these functions `payable` in tests?
I think it's because payable inflicts extra cost, so we calculate on worse case scenario, but just to be on the same page

File: near-sdk/tests/store_performance_tests.rs
Comment (akorchyn) [inline near-sdk/tests/store_performance_tests.rs:28]: I would remove that and instead adjust `perform_asserts` to `fn perform_asserts(total_gas: u64, col: impl Display)` as it's used only for logging purposes. 
It would remove this _ => (col, 0) from other tests. And you can provide your own name for Lazy tests

File: near-sdk/tests/store_performance_tests.rs
Comment (akorchyn) [inline near-sdk/tests/store_performance_tests.rs:481]: Should we also add `get` estimation? As we do for `contains`
Also maybe `insert_take`

**Review by akorchyn — COMMENTED** [review]
Looks good. For some reason, I expected better performance. E.g. LookupSet which has ~1020 inserts compared to LazyOption with ~1200

**Comment by akorchyn** [comment]
@race-of-sloths score 8

**Comment by race-of-sloths** [comment]
@akorchyn Thank you for calling!
    
@ruseinov Thank you for the contribution! Join [Race of Sloths](https://race-of-sloths.com) by simply mentioning me in your comment/PRs description and start collecting Sloth Points through contributions to open source projects.

<details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/ruseinov)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: near-sdk/tests/store_performance_tests.rs
Comment (ruseinov) [reply near-sdk/tests/store_performance_tests.rs:28]: It won't really help the `_ => (col, 0)`, because we have other tests that only test a subset of collections, so I'm not sure it's worth the hassle.

**Comment by ruseinov** [comment]
@akorchyn comments addressed!

**Comment by ruseinov** [comment]
> Looks good. For some reason, I expected better performance. E.g. LookupSet which has ~1020 inserts compared to LazyOption with ~1200

Well, if it has to write on each insert to the storage (via flush) - it makes sense.

File: near-sdk/tests/test-contracts/lazy/src/lib.rs
Comment (akorchyn) [inline near-sdk/tests/test-contracts/lazy/src/lib.rs:63]: I would name it `insert_take` to be consistent with `insert_delete`.

**Review by akorchyn — DISMISSED** [review]
Small nit that you may skip. Awesome

**Comment by ruseinov** [comment]
> Small nit that you may skip. Awesome

I'll need to tune the iterations for the CI, then we are done.. It's a bit of a process, I'll poke you when green.

File: near-sdk/tests/test-contracts/lazy/src/lib.rs
Comment (ruseinov) [reply near-sdk/tests/test-contracts/lazy/src/lib.rs:63]: will do

**Review by akorchyn — APPROVED** [review]

**Pull request merged**
